src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py

```diff
@@ -7,6 +7,24 @@
 
 from azure.cli.core.commands import cli_command
 
+
+def cf_acs(_):
```

Comment:
Yes definitely, I've created `_client_factory.py` w.r.t PR #2110 I'll add this method into it once that PR gets merged and update this PR.